README.md

```diff
@@ -27,23 +29,225 @@ You can use the package as follows:
+
+The unit tests requires valid Salesforce credentials to be executed. The tests are implemented in the way that no new data will remain in the account and no existent data will be modified. However, having Salesforce credentials for sandbox use is recommended.
+
+##### Example of initialization
+
+```python
+from santoku.salesforce import ObjectsHandler
+
+objects_handler = ObjectsHandler(
+    auth_url="your_auth_url",
+    username="your_username",
+    password="your_password",
+    client_id="your_client_id",
+    client_secret="your_client_secret",
+)
+```
+or
+```python
+objects_handler = ObjectsHandler.from_aws_secrets_manager(secret_name="your_salesforce_secret")
+```
+
+##### Examples of insertion with different methods
+
+```python
+contact_payload = {"FirstName": "Alice", "LastName": "Ackerman", "Email": "alice@example.com"}
+
+objects_handler.do_request(method="POST", path="sobjects/Contact", payload=contact_payload)
+```
+or
+```python
+objects_handler.insert_record(sobject="Contact", payload=contact_payload)
+```
+
+##### Example of SOQL
+
+```python
+records = objects_handler.do_query_with_SOQL("SELECT Id, Name from Contact")
+```
+
+### Slack
+
+Slack is a proprietary business communication platform. A Slack Bot is a nifty way to run code and automate tasks. In Slack, a bot is controlled programmatically via a bot user token that can access one or more of Slack’s APIs.
+
+The connection to Slack has been done using [Slack Web API](https://slack.dev/python-slackclient/basic_usage.html)
+
+The use of this subpackage requires having Slack API Token of a Slack Bot, which can be passed as argument in the initializer of the handler class directly, or you can store it in AWS Secrets Manager and retrieve it during the initialization using the `from_aws_secrets_manager` method instead.
+
+This subpackage provide methods to send messages to a channel.
+
+##### Examples of initialization
+```python
+from santoku.slack import SlackBotHandler
+
+slack_bot_handler = SlackBotHandler(api_token="your_api_token")
+```
+or
+```python
+slack_bot_handler = SlackBotHandler.from_aws_secrets_manager(secret_name="your_secret_name")
+```
+
+##### Examples of sending of message
 
-# do something else...
+```python
+slack_bot_handler.send_message(channel="your_chanel_name", message="Your message.")
 ```
 
 ## Development
 
 ### Environment
 
-We provide a development environment that uses Visual Studio Code Remote - Containers extension. This extension lets...
+We provide a development environment that uses Visual Studio Code Remote - Containers extension. This extension lets you use a Docker container in order to have a consistent and easily reproducible development environment.
+
+The files needed to build the container are located in the `.devcontainer` directory.
+* `devcontainer.json` contains a set of configurations and tells how VSCode should access the container.

```

Comment:
of configurations, tells VSCode how to access the container and which extensions it should install.